test_conformance/computeinfo/main.cpp

```diff
@@ -190,6 +190,7 @@ config_info config_infos[] = {
     CONFIG_INFO(1, 1, CL_DEVICE_MEM_BASE_ADDR_ALIGN, cl_uint),
     CONFIG_INFO(1, 1, CL_DEVICE_SINGLE_FP_CONFIG, cl_device_fp_config),
     CONFIG_INFO(1, 1, CL_DEVICE_DOUBLE_FP_CONFIG, cl_device_fp_config),
+    CONFIG_INFO(1, 1, CL_DEVICE_HALF_FP_CONFIG, cl_device_fp_config),
```

Comment:
If this affects the display order, a minor improvement might be to move this line to before the one with `CL_DEVICE_SINGLE_FP_CONFIG` so capabilities are reported in increasing float bit width order.

Reply:
> If this affects the display order

It does.

> so capabilities are reported in increasing float bit width order.

I was following the enum value order, but `CL_DEVICE_DOUBLE_FP_CONFIG` is already breaking that order anyway (and I can see why, as it's nicer to have all *_FP_CONFIG values reported together).  Reporting in increasing float bit width order seems nicer indeed, I'll make the change.